css/components/wine-bottle.css

```diff
@@ -0,0 +1,11 @@
+.wine-bottle {
```

Comment:
In your HTML, you don't need the `p` tag around the `img` tag here.

I'm also wondering relative to which element you want to position your `img` absolutely to, because you have a `relative` property value on your `question-card` and your `question-card__list-item`. 🤔